tests/baselines/reference/intrinsicTypes.symbols

```diff
@@ -7,11 +7,11 @@ type TU2 = Uppercase<'foo' | 'bar'>;  // "FOO" | "BAR"
 >TU2 : Symbol(TU2, Decl(intrinsicTypes.ts, 0, 30))
 >Uppercase : Symbol(Uppercase, Decl(lib.es5.d.ts, --, --))
 
-type TU3 = Uppercase<string>;  // string
+type TU3 = Uppercase<string>;  // Uppercase<string>
 >TU3 : Symbol(TU3, Decl(intrinsicTypes.ts, 1, 36))
 >Uppercase : Symbol(Uppercase, Decl(lib.es5.d.ts, --, --))
 
-type TU4 = Uppercase<any>;  // any
+type TU4 = Uppercase<any>;  // Uppercase<`${any}`>

```

Comment:
Is there a reason we can't just produce `Uppercase<any>`? The choice to qualify it as `${any}` seems to be made here: https://github.com/microsoft/TypeScript/pull/47050/files#diff-d9ab6589e714c71e657f601cf30ff51dfc607fc98419bf72e04f6b0fa92cc4b8R15322

Reply:
Iirc, mostly just because, as written, nongeneric string mapping types can only exist over `string`, template literals, or other mappings (any open-ended stringy type) - it's a bit easier to work with and check for when the type set they're constructed with is smaller.